### PR TITLE
Export Tryit

### LINKED ISSUE #24: Alias `try` as `tryit`
## Problem
Because `try` is a reserved word I find myself doing this a lot `import { try as tryit } from 'radash'`. It's annoying. I shouldn't have to alias it myself considering its obviously a reserved word -- everyone has to alias it (unless they import the whole library `import * as _ from 'radash'`).

## Solution
Add an aliased export to the `src/index.ts` so it can be imported as `tryit`.
```ts
export {
  tryit as try,
  tryit
} from './curry'
```

### PR DESCRIPTION
**Pull request by ahmedhesham6**
- Add TryIt to index.ts
- Add test
Closes #24

### CODE REVIEW

File: src/tests/async.test.ts [hunk 1]
```diff
@@ -207,6 +207,25 @@ describe("async module", () => {
...
+    })
+    test('returns result when no error is thrown', async () => {
+      const [err, result] = await _.tryit(async () => {
+        return 'hello'
+      })()
+      assert.isNull(err)
+      assert.isNotNull(result)
+      assert.equal(result, 'hello')
+    })
+  })
```
Comment (rayepps) [inline src/tests/async.test.ts:228]: The `tryit` function is already covered by the tests. Adding an aliased export wont effect the test coverage so we can remove these.

**Review by rayepps — CHANGES_REQUESTED** [review]
Thanks for the contribution @ahmedhesham6 ! Just one comment on the tests.

File: src/tests/async.test.ts (hunk 1 above)
Comment (ahmedhesham6) [reply src/tests/async.test.ts:228]: but the test coverage fails

Comment (ahmedhesham6) [reply src/tests/async.test.ts:228]: Screen Shot 2022-08-28 at 11 01 34 AM

Comment (ahmedhesham6) [reply src/tests/async.test.ts:228]: The method is tested but the alias is not tested. It is a little bit redundant but needed for coverage

Comment (rayepps) [reply src/tests/async.test.ts:228]: Ah gotcha, I think all we need to do is import it and it should get covered. Can you try adding a one-line test like this?
```ts
test('alias exists', () => { assert.notNull(tryit) } 
```

Comment (ahmedhesham6) [reply src/tests/async.test.ts:228]: Sure

**Review by rayepps — APPROVED** [review]
Perfect @ahmedhesham6, thanks again for doing this!

**Pull request merged**